api/blueprints/drbCollection.py

```diff
@@ -89,6 +89,50 @@ def collectionCreate(user=None):
 
     return APIUtils.formatOPDS2Object(201, opdsFeed)
 
+@collection.route('/update/<uuid>', methods=['POST'])
+@validateToken
+def collectionUpdate(uuid, user=None):
+    logger.info('Updating current collection')
```

Comment:
Maybe we should move this down to after we validate the input? Or change the message to something like "Handling collection update request" since this message might be misleading if the validation below fails?

Reply:
Good suggestion. I'll change the message to reflect what's really going on.